tikv-configuration-file.md

```diff
@@ -1129,11 +1129,6 @@ Configuration items related to RocksDB
 + The maximum RocksDB WAL size in total, which is the size of `*.log` files in the `data-dir`.
 + Default value: `"4GB"`
 
-### `enable-statistics`
```

Comment:
Is `enable-statistics` deprecated or deleted? If it is deprecated, maybe we need to add a warning instead of deleting it from the doc.

Reply:
User will not be able to see it in any way. He can still set it but the value will be ignored.